cpp/include/cudf/detail/null_mask.cuh

```diff
@@ -291,7 +197,7 @@ struct word_num_set_bits_functor {
  * ranges to count the number of set bits within
  * @param indices_end An iterator representing the end of the range of indices specifying ranges to
  * count the number of set bits within
- * @param streaam CUDA stream used for device memory operations and kernel launches
```

Comment:
Same spelling mistake in line 268 as well (old line no 421).

Reply:
Fixed.